mbuild/compound.py

```diff
@@ -1210,22 +1362,23 @@ def energy_minimization(self, steps=2500, algorithm='cg',
                protein models implemented in an off-lattice force field" (2001)
                J. Comput. Chem. 22, 1229-1242
```

Comment:
Since Sphinx won't pick up this docstring, these references should be added to the docstring of `energy_minimization`.

Reply:
Okay I copied the references to `energy_minimize`, should I delete them from `_energy_minimize_openbabel`?